docs/content/principles.mdx

```diff
@@ -3,36 +3,71 @@ name: Principles
 route: /principles
 ---
 
-# Principles
+## Design Principles
```

Comment:
@artsheeps Want to include something here? I know you have something written for this 😏